examples/jsm/loaders/KTX2Loader.js

```diff
@@ -242,33 +242,33 @@ class KTX2Loader extends Loader {
 	load( url, onLoad, onProgress, onError ) {
 
 		if ( this.workerConfig === null ) {
-
-			throw new Error( 'THREE.KTX2Loader: Missing initialization with `.detectSupport( renderer )`.' );
-
+		  throw new Error( 'THREE.KTX2Loader: Missing initialization with `.detectSupport( renderer )`.' );
 		}
-
+	  
 		const loader = new FileLoader( this.manager );
-
+	  
 		loader.setResponseType( 'arraybuffer' );
 		loader.setWithCredentials( this.withCredentials );
-
+	  
 		loader.load( url, ( buffer ) => {
+		  this.parse( buffer, onLoad, onError );
+		}, onProgress, onError );
 
-			// Check for an existing task using this buffer. A transferred buffer cannot be transferred
-			// again from this thread.
-			if ( _taskCache.has( buffer ) ) {
-
-				const cachedTask = _taskCache.get( buffer );
-
-				return cachedTask.promise.then( onLoad ).catch( onError );
```

Comment:
We will need to keep this caching behavior. For example if THREE.Cache is enabled, two requests to the same URL will return the same buffer, and the second request will fail without this cache.